docs/user_guide.md

```diff
@@ -305,3 +305,61 @@ make deploy
     RESOURCE_PREFIX=<kuberntes-resource-name-prefix> \
     WATCH_NAMESPACE=<namespace-to-watch>
 ```
+
+### Cancel running Flink job
+
+If you want to cancel a running Flink job, attach control annotation to your FlinkCluster's metadata:
+```
+metadata:
+  annotations:
+    flinkclusters.flinkoperator.k8s.io/desired-control: job-cancel
```

Comment:
Can we add the command line as well: `kubectl annotate flinkclusters  <name> flinkclusters.flinkoperator.k8s.io/desired-control=job-cancel`?

Reply:
Done.